src/Core/Silk.NET.Core/Miscellaneous/VkNonDispatchableHandle.cs

```diff
@@ -0,0 +1,35 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Runtime.InteropServices;
+
+// TODO when Core 2 is merged in the feature/new-interop branch, find a better place for this struct
```

Comment:
#262 :eyes:

Reply:
I'll merge that one first and then merge in 2.0's changes before merging this PR.